### PR TITLE
Date format is not proper to create a date object in Safari

### PR DESCRIPTION
**Pull request by broccolinisoup**
The details are here : https://projects.engineering.redhat.com/browse/CPFED-2276

### CODE REVIEW

File: src/sidebar/components/timestamp.js [hunk 1]
```diff
@@ -15,14 +15,21 @@ function TimestampController($scope, time) {
   var self = this;
 
   function updateTimestamp() {
-    self.relativeTimestamp = time.toFuzzyString(self.timestamp);
-    self.absoluteTimestamp = dateUtil.format(new Date(self.timestamp));
+    //  This is (should be) a temporary solution.
+    var modifiedTime = self.timestamp.replace(/ /g,'T').concat('Z');
+    /* The 'updated' and 'created' data are sent to server in a format of (YYYY-MM-DDTMM:SS:SS.SSZ)
+    * However, somehow it is returned in a format of (YYYY-MM-DD MM:SS:SS.SS) (No T and Z) and this causes
```
Comment (SiliconValet) [inline src/sidebar/components/timestamp.js]: If this is a bug on the server side, don't fix this - we'll fix it server side.

Comment (broccolinisoup) [reply src/sidebar/components/timestamp.js]: I think it is. I'm going to revert this changes and create a ticket for you.

Comment (SiliconValet) [reply src/sidebar/components/timestamp.js]: I completed this work in the CPDRUPAL ticket. there's n MR out for you to test the branch and validate I haven't broken anything.

**Comment by broccolinisoup** [comment]
This is fixed on the backend so PR is rejected and closed.

**Pull request closed without merging**